### DISCUSSION TITLE
way to download from giy?

### DISCUSSION THREAD

**Discussion by popov-alex** [Binaryify/Material-Theme-vscode#780] [Q&A]
**sorry there is a typo in the topic - the last word is *git

I cannot add this theme to my work computer from the vs code market as it is blocked.

is there a way to download it from git for installation? as was able to do it with material ui like so:

git clone https://github.com/material-theme/vsc-community-material-theme.git Equinusocio.vsc-community-material-theme

thank you!

**Comment by Binaryify** [comment]
1. git clone git@github.com:Binaryify/OneDark-Pro.git
2. yarn
3. yarn package
4. open the vsix file

**Reply by popov-alex** [reply]
Hello!
i get the following error message:
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

**Reply by popov-alex** [reply]
figured it out - problem with ssh
